src/test/java/gov/di_ipv_fraud/step_definitions/FraudAPIStepDefs.java

```diff
@@ -76,7 +75,7 @@ private String getPrivateAPIEndpoint(){
             throw new IllegalArgumentException("Environment variable PRIVATE API endpoint is not set");
         }
         LOGGER.info("privateAPIEndpoint =>"+privateAPIEndpoint);
-        return  "https://" + privateAPIEndpoint + ".execute-api.eu-west-2.amazonaws.com/build";
+        return  "https://" + privateAPIEndpoint + ".execute-api.eu-west-2.amazonaws.com/dev";
```

Comment:
Same here

Reply:
This is now being passed as a parameter.